src/idsk/overrides/_assets.scss

```diff
@@ -11,21 +11,21 @@
 /// @type String
 /// @access public
 
-$govuk-assets-path: "/assets/" !default;
+$idsk-assets-path: "/assets/" !default;
 
 /// Path to the images folder, with trailing slash.
 ///
 /// @type String
 /// @access public
 
-$govuk-images-path: "#{$govuk-assets-path}images/" !default;
+$govuk-images-path: "#{$idsk-assets-path}images/" !default;
```

Comment:
tu nedame tiez $idsk-image-path?

Reply:
To by si následne aj pýtalo meniť "$govuk-image-url-function" a povedal som si, že task sa týka iba fontov..
Ale okay, idem teda popremenovávať aj veci týkajúce sa images ciest. Bude to jedine lepšie.